etc/config/c++.amazon.properties

```diff
@@ -1223,6 +1223,15 @@ libs.expected_lite.versions.010.version=0.1.0
 libs.expected_lite.versions.010.path=/opt/compiler-explorer/libs/expected-lite/v0.1.0/include
 libs.expected_lite.versions.trunk.version=trunk
 libs.expected_lite.versions.trunk.path=/opt/compiler-explorer/libs/expected-lite/trunk/include
+libs.highway.name=Highway
+libs.highway.versions=trunk:0.12.2
```

Comment:
This _might_ work; but periods in names are used to resolve the sub-components of our (crappy) config system. I'm going to merge, but change this to 0_12_2 for consistency (and honestly, safety of my naff code)